package.json

```diff
@@ -7,7 +7,6 @@
     "igv": "2.1.0",
     "jquery": "3.4.0",
     "jquery-ui": "1.12.1",
-    "lodash": "4.17.11",
```

Comment:
I was curious where we use Lodash, and per https://github.com/broadinstitute/single_cell_portal_core/commit/77a867413a5ed8aefb6c1f68a2941605fd036aaf it seems we introduced it to address a separate CVE.

Reply:
correct - just removing an unnecessary peer dependency.